helm/g8s-oauth2-proxy-chart/templates/deployment.yaml

```diff
@@ -16,14 +16,18 @@ spec:
         k8s-app: oauth2-proxy
     spec:
       containers:
-        # Use bitly fork from https://github.com/giantswarm/oauth2_proxy
-      - image: quay.io/giantswarm/oauth2_proxy:38072cbaa6abfc6e9f647c1ee0e4c7afa5ba194e
+        # Use bitly fork from https://github.com/pusher/oauth2_proxy/tree/kubernetes,
+        # which supports OICD token renewal and has ca certs installed
+      - image: {{ .Values.Installation.V1.Registry.Domain }}/giantswarm/oauth2_proxy:kubernetes-3-ca-certs
         name: oauth2-proxy
         args:
-        - --cookie-expire=0h30m0s
+        - --cookie-expire=24h0m0s
+        - --cookie-refresh=0h60m0s
```

Comment:
Tokens get auto-refreshed now. Do same with cookies: valid for a day, refresh after an hour (so its valid another day)

Reply:
Is this `kubernetes-3-ca-certs` tag name significant? Do we need to update the tag when updating the image?

Reply:
No, its just the retagged "version" of the original image. (Unfortunatly) the retagger does not allow renaming tags.